tests/example.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Example
```

Comment:
come to think of it, do we really need Gherkin in this project? The library is pretty technical, tests written in C++ should describe it's behavior quite succinctly, and the target audience will be able to read them.

Reply:
It is not a matter of target audience - the main concept is to describe functionality with natural language, to combine documentation and tests tightly because in another way you need to interpret test and sometimes it can be not so easy to do it. The natural language description is more user-friendly than a bunch of tests with very weird names in most cases. So this is all about functional or acceptance tests. Of course, it is not necessary to use Gherkin for unit testing such us serialization mechanism or query config parsing functionality.

Reply:
We can use Gherkin (GUnit allows), if we find reason, but for now I will remove example. It still available [here](https://github.com/cpp-testing/GUnit/blob/master/example/GScenario.feature).